### PR TITLE
fix: double tapping input selects entire string

### LINKED ISSUE #2298: Double tapping input box doesn't select entire string
**Describe the bug**
Double tapping inside destination input box does not select entire string.

**To Reproduce**

- video attached in screenshots section

**Expected behavior**
Entire string should be selected when double tapping the string inside the input box.

**Screenshots**

https://github.com/GaloyMoney/galoy-mobile/assets/17320781/ccefb7d3-2285-4bde-b28d-061c198af3ca

**Smartphone (please complete the following information):**

- Device: S21 5G
- OS: Android 12

### PR DESCRIPTION
**Pull request by siiddhantt**
fixes #2298

- added a function to handle double-taps

https://github.com/GaloyMoney/galoy-mobile/assets/47355538/c9a146fc-4112-42b3-9a90-dc4c3f7b9842

### CODE REVIEW

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (nicolasburtey) [inline app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:1]: we dont want any `any` in our codebase

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 2]
```diff
@@ -205,10 +212,26 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
         payload: { unparsedDestination: newDestination },
       })
       setGoToNextScreenWhenValid(false)
+      setTextInputSize(newDestination.length)
     },
     [dispatchDestinationStateAction, setGoToNextScreenWhenValid],
   )
 
+  const handleDoubleClick = () => {
```
Comment (nicolasburtey) [inline app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:220]: `tap` is more appropriate than `click` when it's mobile related

**Comment by nicolasburtey** [comment]
thanks for the contribution. few questions/feedback.

I didn't check the API from TexttInput but is there no native API to handle long press natively?

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 1 above)
Comment (siiddhantt) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:1]: sure, will remove that!

**Comment by siiddhantt** [comment]
> thanks for the contribution. few questions/feedback.
> 
> I didn't check the API from TexttInput but is there no native API to handle long press natively?

no there is no native way to handle double-taps or long press afaik.

**Comment by nicolasburtey** [comment]
> > thanks for the contribution. few questions/feedback.
> > I didn't check the API from TexttInput but is there no native API to handle long press natively?
> 
> no there is no native way to handle double-taps or long press afaik.

we quickly discuss yesterday with the team and it seems there is a proper parameter to activate that would handle the double tap properly. don't have the exact parameters in mind, you can maybe look into the documentation, otherwise someone on the team will have a look later

**Comment by MaxwellDG** [comment]
Merged and readdressed in #2920 with a slightly different solution

**Pull request closed without merging**